docs/guidelines-for-reviewing-datascience-code.md

```diff
@@ -0,0 +1,15 @@
+# Guidelines for Reviewing Data Science Code
+
+Code Reviews are a good way to share knowledge within a team, especially when teammates are working on different projects and issues. This helps to ensure exposure to new ideas and technologies, quality check, standardize the coding process and knowledge sharing across the code and the team. 
+
+1. **Understand the PR** : When asked for a review, make sure to first understand the issue linked and the goal of the pull request.
+2. **NB Review** : Use tools like NB review when possible while reviewing Jupyter Notebooks, this will make reviewing the notebooks easier.
+3. **Don’t be afraid to ask questions** : When reviewing someone’s code, if you don't understand something or are curious about why a certain approach was implemented, ask questions and get clarification. 
+4. **Read Carefully** : Read the notebook thoroughly and try to look for logical errors or bugs in the code. Make sure all edge cases are taken care of.
+5. **Find ways to clean and condense the code** : Suggest alternate approaches to perform a similar action as the current code. This will present new techniques and various implementations for the team to consider.
+6. **Future steps and actions** : Make sure the goals of the notebook are met in the end, the key takeaways are highlighted and should contain a layer of interpretation and abstraction of statistical results. While reviewing the code, it is important to ensure that future actions are being discussed so as to enable the teammates to envision for the future.
```

Comment:
```suggestion
6. **Future steps and actions** : Make sure the goals of the notebook are met in the end, the key takeaways are highlighted and should contain the data scientist's interpretation of any statistical and/ or experimental results. While reviewing the code, it is important to ensure that future actions are mentioned so that readers or future developers picking up the work will know the developer's intentions for follow up experiments or implementations. 
```